docs/user-guide/ingest-data/for-iot/influxdb-line-protocol.md

```diff
@@ -68,9 +137,64 @@ To configure Telegraf, simply add GreptimeDB URL into Telegraf configurations:
 
 ## Data model
 
-To understand the differences between the data models of InfluxDB and GreptimeDB, please refer to the [data model in differences](/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md#data-model-in-difference) section in the Migrate from InfluxDB to GreptimeDB document.
+While you may already be familiar with [InfluxDB key concepts](https://docs.influxdata.com/influxdb/v2/reference/key-concepts/), the [data model](/user-guide/concepts/data-model.md) of GreptimeDB is something new to explore.
+Here are the similarities and differences between the data models of GreptimeDB and InfluxDB:
+
+- Both solutions are [schemaless](/user-guide/ingest-data/overview.md#automatic-schema-generation), eliminating the need to define a schema before writing data.
+- In InfluxDB, a point represents a single data record with a measurement, tag set, field set, and a timestamp.
+In GreptimeDB, it is represented as a row of data in the time-series table,
+where the table name aligns with the measurement,
+and the columns are divided into three types: Tag, Field, and Timestamp.
+- GreptimeDB uses `TimestampNanosecond` as the data type for timestamp data from the [InfluxDB line protocol API](/user-guide/ingest-data/for-iot/influxdb-line-protocol.md).
+- GreptimeDB uses `Float64` as the data type for numeric data from the InfluxDB line protocol API.
```

Comment:
Another key point is that greptimedb tables for InfluxDB will set the `'merge_mode'='last_non_null'` table option to keep the behavior the same as InfluxDB. It's worth to be noted here.